openrr-command/src/robot_command.rs

```diff
@@ -328,11 +328,13 @@ impl RobotCommandExecutor {
                 x,
                 y,
                 theta,
-                wait_secs,
+                duration_secs,
             } => {
-                client.send_velocity(&BaseVelocity::new(*x, *y, *theta))?;
-                // wait publish
-                sleep(Duration::from_secs_f64(*wait_secs));
+                let start = Instant::now();
+                while start.elapsed() < Duration::from_secs_f64(*duration_secs) {
```

Comment:
This is a small improvement, but i want to move `Duration::from` out of the loop. 
`Duration::from_secs_f64(*duration_secs)` is called so many times (for example, 300times for 3secss).
and L336.